### ISSUE TITLE
[Feature request] Add setting for currency

### ISSUE THREAD

**Issue by Huy-Ngo** [GittyMac/WalletCount#10]
Currently the currency is determined by locale, which is unfortunately unseparable from language on Android. As some people prefer to use languages different from their currency, I think it would be nice to add a setting for that.

**Comment by NRG-600** [comment]
> Currently the currency is determined by locale, which is unfortunately unseparable from language on Android. As some people prefer to use languages different from their currency, I think it would be nice to add a setting for that.

Yup that's right there should be an option to choose preferred currency

**Comment by WarmMarbles** [comment]
I feel like a good approach would be to both add a button to remove decimals (since some countries don't make use of decimals and only count in whole numbers) and an option to change the currency symbol.

**Comment by Hugo22O** [comment]
I just want to underline this issue. I'm in Europe and have my locale set to English (UK), yet I use the Euro as currency.
